### PR TITLE
adding user docs how to access metrics in Openshift

### PR DESCRIPTION
**Pull request by wejdross**

### CODE REVIEW

**Comment by lieneluksika** [comment]
@akosma could you please take a look at the docs, does it make sense to you?

File: docs/modules/ROOT/pages/vshn-managed/how-tos/access_metrics.adoc [hunk 1]
```diff
@@ -0,0 +1,13 @@
+= Access VSHN Managed Services metrics in Openshift console
+
+You can browse VSHN Managed Services metrics in Openshift console. To do that, please follow below steps:
```
Comment (lieneluksika) [inline docs/modules/ROOT/pages/vshn-managed/how-tos/access_metrics.adoc:3]:
```suggestion
You can browse metrics for VSHN Managed Services in the Openshift console. To do that, please follow these steps:
```

**Comment by lieneluksika** [comment]
yes, visuals are a good idea

**Comment by wejdross** [comment]
added 2 screenshots that I think are most important for our customers, but I can't force first screenshot to have normal dimensions - antora docs says how to do that, but it doesn't work for me :/

**Pull request merged**